configuration/buildHelper.py

```diff
@@ -98,10 +98,10 @@ def main(factoryFile,package,buildDir):
 #    import isce
     import filecmp
     try:
-        import imp
+        import importlib
         factoryFile = os.path.abspath(factoryFile)
-        mod = imp.find_module(factoryFile.replace('.py',''))
-        factModule = imp.load_module(factoryFile.replace('.py',''),mod[0],mod[1],mod[2])
+        mod = importlib.util.find_spec(factoryFile.replace('.py',''))
```

Comment:
```suggestion
        mod = importlib.util.spec_from_file_location('.', factoryFile)
```

This should fix the helper errors.